### PR TITLE
Added Prims Algorithm

### PR DESCRIPTION
**Pull request by Subhojit-Kumar-Saha**
/***** Prim's Algorithm for finding Minimum Spanning Tree *****/

Sample Inputs are already given :

graph[V][V] = { { 0, 2, 0, 6, 0 },
{ 2, 0, 3, 8, 5 },
{ 0, 3, 0, 0, 7 },
{ 6, 8, 0, 0, 9 },
{ 0, 5, 7, 9, 0 } };

Sample Output:

Edge Value
1 - 0 2
2 - 1 3
3 - 0 6
4 - 1 5

### CODE REVIEW

File: prims_algorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#include <bits/stdc++.h>
+using namespace std;
```
Comment (Bhavnaharitsa) [inline prims_algorithm.cpp:2]: Add algorithm name, sample description, input output of tha algorithm in comments

File: prims_algorithm.cpp [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    }
+
+
+    for(int u = 1; u<V; u++)
+    {
+        for(int i=0;i<V;i++)
+        {
+            if(graph[i][u]&&mstSet[i] == false && graph[i][u]<key[u])
+            {
+                key[u] = graph[i][u];
```
Comment (Bhavnaharitsa) [inline prims_algorithm.cpp]: No single letter variable

File: prims_algorithm.cpp (hunk 1 above)
Comment (Subhojit-Kumar-Saha) [reply prims_algorithm.cpp:2]: /***** Prim's Algorithm for finding Minimum Spanning Tree  *****/

Sample Inputs are already given :
graph[V][V] = { { 0, 2, 0, 6, 0 },
                        { 2, 0, 3, 8, 5 },
                        { 0, 3, 0, 0, 7 },
                        { 6, 8, 0, 0, 9 },
                        { 0, 5, 7, 9, 0 } };

Sample Output: 

Edge    Value
1 - 0      2
2 - 1      3
3 - 0      6
4 - 1      5

Comment (Bhavnaharitsa) [reply prims_algorithm.cpp:2]: not here comments I meant in the code..comment the input output and the idea of algorithm in the code not here

**Pull request closed without merging**